From libc-alpha-return-35113-listarch-libc-alpha=sources dot redhat dot com at sourceware dot org Mon Dec 03 09:38:08 2012
Return-Path: <libc-alpha-return-35113-listarch-libc-alpha=sources dot redhat dot com at sourceware dot org>
Delivered-To: listarch-libc-alpha at sources dot redhat dot com
Received: (qmail 4372 invoked by alias); 3 Dec 2012 09:38:05 -0000
Received: (qmail 3968 invoked by uid 22791); 3 Dec 2012 09:38:03 -0000
X-SWARE-Spam-Status: No, hits=-2.1 required=5.0
	tests=AWL,BAYES_00,RP_MATCHES_RCVD
X-Spam-Check-By: sourceware.org
From: Allan McRae <allan at archlinux dot org>
To: libc-alpha at sourceware dot org
Subject: [PATCH] Remove @hsep and @vsep usage from info pages
Date: Mon,  3 Dec 2012 19:37:44 +1000
Message-Id: <1354527464-15109-1-git-send-email-allan@archlinux.org>
Mailing-List: contact libc-alpha-help at sourceware dot org; run by ezmlm
Precedence: bulk
List-Id: <libc-alpha.sourceware.org>
List-Subscribe: <mailto:libc-alpha-subscribe at sourceware dot org>
List-Archive: <http://sourceware.org/ml/libc-alpha/>
List-Post: <mailto:libc-alpha at sourceware dot org>
List-Help: <mailto:libc-alpha-help at sourceware dot org>, <http://sourceware dot org/ml/#faqs>
Sender: libc-alpha-owner at sourceware dot org
Delivered-To: mailing list libc-alpha at sourceware dot org

2012-12-03  Allan McRae  <allan@archlinux.org>

	* manual/stdio.texi (Predefined Printf Handlers): Remove
	@hsep and @vsep usage.

Upstream-Status: Backport
---

As far as I can tell, @hsep and @vsep have not been supported in texinfo for
a long time, although it attempted to add the requested separators resulting
in ugliness:

     +------+--------------+--------+--------+---------------+
     |low|Multiplier|From|Upper|Multiplier|
     |' '|1||' '|1|
     |k|2^10 (1024)|kilo|K|10^3 (1000)|

Go to "info libc" then 
IO on streams -> Customizing Printf -> Predefined Printf Handlers
to see this section.

With texinfo 4.13.91, this becomes an error:

./stdio.texi:3140: Unknown command `hsep'
./stdio.texi:3140: Unknown command `vsep'

There is also 215 lines of warnings to be fixed at a later date...


 manual/stdio.texi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manual/stdio.texi b/manual/stdio.texi
index be769a5..7b436f0 100644
--- a/manual/stdio.texi
+++ b/manual/stdio.texi
@@ -3137,7 +3137,7 @@ The postfix tag corresponds to bytes, kilobytes, megabytes, gigabytes,
 etc.  The full table is:
 
 @ifinfo
-@multitable @hsep @vsep {' '} {2^10 (1024)} {zetta} {Upper} {10^24 (1000)}
+@multitable {' '} {2^10 (1024)} {zetta} {Upper} {10^24 (1000)}
 @item low @tab Multiplier  @tab From  @tab Upper @tab Multiplier
 @item ' ' @tab 1           @tab       @tab ' '   @tab 1
 @item k   @tab 2^10 (1024) @tab kilo  @tab K     @tab 10^3 (1000)
-- 
1.8.0.1

